modules/core/src/main/scala/ch/epfl/bluebrain/nexus/admin/core/config/AppConfig.scala

```diff
@@ -43,7 +43,7 @@ object AppConfig {
 
   final case class DescriptionConfig(name: String, environment: String) {
     val version: String = BuildInfo.version
-    val ActorSystemName = s"$name-${version.replaceAll("\\.", "-")}-$environment"
+    val actorSystemName = s"$name-${version.replaceAll("\\.", "-")}-$environment"
```

Comment:
I would suggest removing the environment

Reply:
Also since we switched to sbt-release-early, there might be other symbols in the version string when you build a snapshot.

I've used `version.replaceAll("\\W", "-")` myself.